pkg/resolution/resolver/git/resolver.go

```diff
@@ -323,17 +362,20 @@ func (r *resolvedGitResource) Annotations() map[string]string {
 	if r.Repo != "" {
 		m[AnnotationKeyRepo] = r.Repo
 	}
-	if r.URL != "" {
-		m[AnnotationKeyURL] = r.URL
-	}
 
 	return m
 }
 
 // Source is the source reference of the remote data that records where the remote
 // file came from including the url, digest and the entrypoint.
 func (r *resolvedGitResource) Source() *pipelinev1beta1.ConfigSource {
-	return nil
+	return &pipelinev1beta1.ConfigSource{
+		URI: spdxGit(r.URL),
+		Digest: map[string]string{
+			"sha1": r.Revision,
+		},
+		EntryPoint: r.Path,
+	}
```

Comment:
Hi @wlynch ,
For the url field, I am trying to format the url to the spdx download format as recommended by intoto [here](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#:~:text=ResourceURI%20(string)). Do you think we should do it as is in pipeline or we should format it in Chains?

Please let me know if you have any concerns or feedback. Thanks you!!!

Reply:
Fine with me!